src/Controls/samples/Controls.Sample/Startup.cs

```diff
@@ -30,7 +30,9 @@ public void Configure(IAppHostBuilder appBuilder)
 				appBuilder = appBuilder
 					.UseMauiApp<MyApp>();
 			}
-
+#if DEBUG
```

Comment:
the decision to enable HotReload or not shouldn't be left to end users, or we would end up with HR being enabled on releases build

Reply:
That should be fine, if they wanted it. If we remove this. Then we cannot hot reload iOS, and Android at the same time. Which currently works.